package.json

```
@@ -52,7 +52,7 @@
     "webpack-cli": "^3.3.10"
   },
   "bugs": "https://github.com/creativecommons/vocabulary/issues/",
-  "homepage": "https://creativecommons.github.io/vocabulary/",
+  "homepage": "https://opensource.creativecommons.org/vocabulary",
```

If you change it to this, you wouldn't even need the redirect?
```suggestion
  "homepage": "https://opensource.creativecommons.org/cc-vocabulary/",
```

---

https://opensource.creativecommons.org/vocabulary is already live and working too. the redirects for `/vocabulary` and `/cc-vocabulary` are the same.